### PR TITLE
Soaring: Add airspeed management/speed-to-fly documentation.

### PR DESCRIPTION
**Pull request by samuelctabor**
Documentation for new features introduced in https://github.com/ArduPilot/ardupilot/pull/17650

### CODE REVIEW

File: plane/source/docs/soaring-upcoming.rst [hunk 1]
```diff
@@ -153,6 +153,25 @@ If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIU
 
 .. _soaring_maximum-distance-from-home:
 
+
+Airspeed Management (Optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is also possible to let soaring control the airspeed in certain situations.
+
+In THERMAL mode, the parameter :ref:`SOAR_THML_ARSPD<SOAR_THML_ARSPD>` set the target airspeed in metres per second.
```
Comment (nkruzan) [inline plane/source/docs/soaring-upcoming.rst:162]:
```suggestion
In THERMAL mode, the parameter :ref:`SOAR_THML_ARSPD<SOAR_THML_ARSPD>` sets the target airspeed in metres per second.
```

File: plane/source/docs/soaring-upcoming.rst [hunk 2]
```diff
@@ -153,6 +153,25 @@ If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIU
...
+
+In AUTO, FBWB and CRUISE modes, the parameter :ref:`SOAR_CRSE_ARSPD<SOAR_CRSE_ARSPD>` controls the target airspeed when the aircraft is gliding.
+
+ - Set :ref:`SOAR_CRSE_ARSPD<SOAR_CRSE_ARSPD>` to a positive number in metres per second to use this as the constant target airspeed.
+
+ - Set :ref:`SOAR_CRSE_ARSPD<SOAR_CRSE_ARSPD>` to -1 to automatically calculate the best airspeed based on speed-to-fly theory. The best airspeed is based on current estimated lift/sink, headwind and the setting of :ref:`SOAR_VSPEED<SOAR_VSPEED>`. See the below talk for more information.
+
+If one or the other of the above parameters are zero, or when the aircraft is using power (throttle > 0), :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` is used in the corresponding flights modes as usual.
```
Comment (nkruzan) [inline plane/source/docs/soaring-upcoming.rst:170]:
```suggestion
If one or the other of the above parameters are zero, or when the aircraft is using power (throttle > 0), :ref:`TRIM_ARSPD_CM<TRIM_ARSPD_CM>` is used in the corresponding flight modes as usual.
```

**Comment by samuelctabor** [comment]
Thanks @nkruzan

**Comment by samuelctabor** [comment]
Added to upcoming features list.

File: common/source/docs/common-master-features.rst [hunk 3]
```diff
@@ -21,6 +21,7 @@ This lists features or changes merged in master and build in the "latest" firmwa
     MSP Telemetry Addition of Canvass Mode <common-msp-overview-4.2>
     Packet Digital MPPT Solar Controller <common-packetdigital-mppt>
     Swan K-1 Copter Tailsitter <common-Swan-K1>
+    Soaring upcoming features (plane only) <soaring-upcoming>
```
Comment (Hwurzburg) [inline common/source/docs/common-master-features.rst:24]:
```suggestion
[site wiki="plane']
    Soaring upcoming features <soaring-upcoming>
[/site]
```

File: plane/source/docs/soaring-upcoming.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.. _soaring-upcoming:
```
Comment (Hwurzburg) [inline plane/source/docs/soaring-upcoming.rst:1]:
```suggestion
.. _soaring-speed-to-fly:
```
need to change file name to this also

Comment (Hwurzburg) [reply plane/source/docs/soaring-upcoming.rst:1]: Need to add a note in common-future-wiki-changes in Plane section for the reference to this page that will be added to the soaring page

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: plane/source/docs/soaring-upcoming.rst (hunk 4 above)
Comment (samuelctabor) [reply plane/source/docs/soaring-upcoming.rst]: Ok check now.

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**